includes/class-freemius.php

```diff
@@ -5273,7 +5693,7 @@ function stop_site_tracking() {
             }
 
             // Send update to FS.
-            $result = $this->get_api_site_scope()->call( '/?fields=is_disconnected', 'put', array(
+            $result = $this->get_api_site_scope()->call( $this->add_is_clone_param( '/?fields=is_disconnected' ), 'put', array(

```

Comment:
@fajardoleo instead explicitly adding `$this->add_is_clone_param()`, from a code design perspective, it's better to create a helper method(s) that will wrap `$this->get_api_site_scope()->call()` and `$this->get_api_site_scope()->get()` and use those. That way, we won't need to remember it when adding new calls in the future + if we would want to change something in the logic that maybe impacts more than just the querystring param, we'll be able to make a change in one (or two) places.